assets/terraform-modules/aws/flatcar-linux/kubernetes/workers/cl/worker.yaml.tmpl

```diff
@@ -5,7 +5,11 @@ systemd:
       enable: true
     - name: iscsid.service
       enable: true
-      enabled: true
```

Comment:
nit: seems like `enable`/`enabled` duplication could be removed in a separate commit.

Reply:
Added the change info to the commit.